kubernetes/validators.go

```diff
@@ -181,25 +181,6 @@ func validatePositiveInteger(value interface{}, key string) (ws []string, es []e
 	return
 }
 
-func validateDNSPolicy(value interface{}, key string) (ws []string, es []error) {
```

Comment:
These are more functions that are unused in the code base, so I cleaned them up.